### DISCUSSION TITLE
Ucdn.php failed to open stream: No such file or directory

### DISCUSSION THREAD

**Discussion by KawsarAhmad43** [mpdf/mpdf#1920] [Q&A]
### Guidelines

- [X] I understand that [if I fail to provide all required details, this issue may be closed without review](https://github.com/mpdf/mpdf/blob/development/.github/CONTRIBUTING.md).

### Description of the bug

Hi 
In my App there is a option to download profile. When i'm clicking on the button to download it says:[cpanel]
include(/home/myApp/public_html/vendor/composer/../mpdf/mpdf/src/Ucdn.php): failed to open stream: No such file or directory

### mPDF version

^8.0.0

### PHP Version and environment (server type, cli provider etc., enclosing libraries and their respective versions)

^7.4|^8.0

### Reproducible PHP+CSS+HTML snippet suffering by the error

```php
$pdf = PDF::loadView( 'pdf.profile', compact( 'profile') );
return $pdf->download( 'profile.pdf' );
```

**Comment by Muhammad-Sarfaraz** [comment]
Did you got any solutions, yet?

**Reply by KawsarAhmad43** [reply]
@Muhammad-Sarfaraz  no sir
